src/Neos.NeosIo.FeatureList/Resources/Private/Fusion/Integration/Content/FeatureList.fusion

```diff
@@ -0,0 +1,66 @@
+prototype(Neos.NeosIo.FeatureList:Content.FeatureList) < prototype(Neos.Neos:ContentComponent) {
+
+	featureGroups = Neos.Fusion:RawCollection {
+		collection = ${q(node).children('[instanceof Neos.NeosIo.FeatureList:Content.FeatureGroup]')}
+		itemName = 'featureGroup'
+		itemRenderer = Neos.Fusion:RawArray {
+			__node = ${featureGroup}
```

Comment:
Could you explain what `__node` does here?

Reply:
__node is used to apply contentElementWrapping below. This is needed to properly interact with this node hierarchy.